schemas.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from datetime import date
+from typing import Optional
+from pydantic import BaseModel, Field
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Authors(AuthorBase):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class AuthorUpdate(AuthorBase):
+    name: Optional[str] = None
+    bio: Optional[str] = None
+
+    class Config:
+        from_attributes = True
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    author_id: int = Field(..., gt=0)
+
+
+class Books(BookBase):
```

Comment:
Consider renaming the `Books` class to `Book` to maintain consistency with naming conventions and the rest of the codebase.